src/app/payment/views/CheckoutSuccessView/CheckoutSuccessView.tsx

```diff
@@ -10,7 +10,7 @@ const CheckoutSuccessView = (): JSX.Element => {
 
   useEffectAsync(async () => {
     await dispatch(userThunks.refreshUserThunk());
-    analyticsService.trackPaymentConversion();
+    await analyticsService.trackPaymentConversion();
```

Comment:
do not make the checkout depend on analytics

Reply:
it has a try catch inside though 🤔

Reply:
It would be better not to rely on service as the component is responsible for its own behavior. So if the component has to handle the checkout success and sending the analytics is not critical for handling the success is the component who is responsible for ensuring that behavior really happens as expected. 

Therefore, if someone decides to remove the `try-catch`, the side effects are avoided. 

'The checkout is suddenly broken'. Then you find that someone removed something non-directly related to the checkout but still break it, which is very serious. 

In summary, a good idea will be to extract this function from the hook if possible and add a `try-catch` there to ensure that the component accomplishes its responsibilities always.